package.json

```diff
@@ -1,130 +1,125 @@
-    "fs-extra": "^7.0.0",
-    "grunt": "^1.0.2",
-    "grunt-babel": "^7.0.0",
-    "grunt-contrib-clean": "^2.0.0",
-    "grunt-contrib-concat": "^1.0.1",
-    "grunt-contrib-connect": "^2.0.0",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-uglify": "^4.0.0",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-eslint": "^21.0.0",
-    "grunt-markdownlint": "^2.0.0",
-    "grunt-mocha": "1.1.0",
-    "grunt-parallel": "^0.5.1",
-    "grunt-retire": "^1.0.7",
-    "html-entities": "^1.2.0",
-    "husky": "^1.1.1",
-    "jquery": "^3.0.0",
-    "jsdoc": "^3.5.5",
-    "lint-staged": "^8.0.0",
-    "markdown-table": "^1.1.2",
-    "minami": "^1.2.3",
-    "mocha": "^5.2.0",
-    "prettier": "^1.14.0",
-    "promise": "~8.0.1",
-    "revalidator": "~0.3.1",
-    "selenium-webdriver": "~3.6.0",
-    "sri-toolbox": "^0.2.0",
-    "standard-version": "^4.4.0",
-    "typescript": "^2.9.2",
-    "uglify-js": "^3.4.4"
-  },
-  "dependencies": {},
-  "lint-staged": {
-    "*.js": [
-      "prettier --write",
-      "git add"
-    ],
-    "*.ts": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  }
+	"name": "axe-core",
+	"description": "Accessibility engine for automated Web UI testing",
+	"version": "3.1.2",
+	"license": "MPL-2.0",
+	"engines": {
+		"node": ">=4"
+	},
+	"contributors": [
+		{
+			"name": "David Sturley",
+			"organization": "Deque Systems, Inc.",
+			"url": "http://deque.com/"
+		},
+		{
+			"name": "Dylan Barrell",
+			"email": "dylan@barrell.com",
+			"organization": "Deque Systems, Inc.",
+			"url": "http://deque.com/"
+		},
+		{
+			"name": "Wilco Fiers",
+			"organization": "Deque Systems, Inc.",
+			"url": "http://deque.com/"
+		},
+		{
+			"name": "Dian Fay",
+			"organization": "Deque Systems, Inc.",
+			"url": "http://deque.com/"
+		},
+		{
+			"name": "Marcy Sutton",
+			"organization": "Deque Systems, Inc.",
+			"url": "http://deque.com/"
+		}
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/dequelabs/axe-core.git"
+	},
+	"keywords": [
+		"Accessibility",
+		"a11y",
+		"testing",
+		"unit",
+		"tdd",
+		"bdd",
+		"aXe"
+	],
+	"main": "axe.js",
+	"typings": "axe.d.ts",
+	"standard-version": {
+		"scripts": {
+			"postbump": "npm run sri-update"
+		}
+	},
+	"scripts": {
+		"api-docs": "jsdoc --configure .jsdoc.json",
+		"build": "grunt",
+		"test-dts": "tsc",
+		"test": "npm run test-dts && grunt test",
+		"test-fast": "grunt test-fast",
+		"version": "echo \"use 'npm run release' to bump axe-core version\" && exit 1",
+		"prepublishOnly": "grunt build && grunt file-exists",
+		"release": "standard-version -a",
+		"next-release": "standard-version --scripts.prebump=./build/next-version.js --skip.commit=true --skip.tag=true",
+		"sri-update": "grunt build && node build/sri-update && git add sri-history.json",
+		"fmt": "prettier --write *.{json,md,js} **/*.ts '{build,doc,lib,test}/**/*.{json,md,js,ts}'"
```

Comment:
`**/*.ts` is probably too greedy. We should avoid directories like `node_modules`.

Reply:
Added a `.prettierignore` for `node_modules`. Good spot.